src/util/dev/server.ts

```diff
@@ -423,17 +451,17 @@ export default class DevServer {
 
     const pkg = await this.getPackageJson();
 
-    // The default empty `now.json` is used to serve all files as static
-    // when no `now.json` is present
-    let config: NowConfig = this.cachedNowConfig || { version: 2 };
-
     // We need to delete these properties for zero config to work
     // with file changes
     if (this.cachedNowConfig) {
       delete this.cachedNowConfig.builds;
       delete this.cachedNowConfig.routes;
     }
 
+    // The default empty `now.json` is used to serve all files as static
+    // when no `now.json` is present
+    let config: NowConfig = this.cachedNowConfig || { version: 2 };
```

Comment:
Why was this moved down?

Reply:
@styfle I moved the `delete` statements up, to make it more clear that when `this.cachedNowConfig` is used the `builds` and `routes` property are not available anymore